### ISSUE TITLE
Pandas dataframes with multi-level column index are not supported by flowkit.Sample()

### ISSUE THREAD

**Issue by andreas-wilm** [whitews/FlowKit#62]
Hello again,

**Describe the bug**

I have a sample with PnN and PnS and when exported via `flowkit.Sample.as_dataframe()` this shows up correctly as multi-level column index, e.g. `MultiIndex([('Bi209Di', 'CD16'),...], names=['pnn', 'pns'])`. In theory I should be able to use the dataframe to instantiate a Sample with `flowkit.Sample(fcs_path_or_data, channel_labels=None...)` where according to the docs FCS data can be
> Pandas DataFrame containing FCS event data (channel labels as column labels)

However, this fails with `AttributeError: 'tuple' object has no attribute 'replace'`.

After conversion to a single level column index, this works as expected, i.e. `df.columns = df.columns.droplevel()` is required.

**Expected behavior**

Ideally multi-level column indices should be supported by `flowkit.Sample()`, but since this is not critical, I thought this behaviour could at least be captured in the examples or added to the docs as caveat.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Python version 3.8.5
 - FlowKit version develop cd4d40f4b69d18ccd2931a68d3e019476f2c686c

Many thanks!

**Comment by whitews** [comment]
Hey Andreas,

Thank you (again!), this is a good find. The Sample class should definitely handle this case since it creates that type of DataFrame. 

I've added support for this in the latest dev commits. I decided to test this case by checking the number of column levels instead of checking for a MultiIndex instance...the rationale being that it doesn't matter whether it's a MultiIndex but  whether there are at least 2 levels. Can verify this is working for your use case? If it does, feel free to close this issue.
